go/mysql/mysql.go

```diff
@@ -38,6 +38,87 @@ func init() {
 }
 
 const (
+	// TypeTiny specifies a TINYINT type.
```

Comment:
For proto3, I think we should come up with our own system that is more consistent. Having to look at types some times, and flags some times is confusing.
This is only the first step towards that formalization.